packages/teleport-publisher-vercel/src/browser.ts

```diff
@@ -12,8 +12,8 @@ export const getSHA = async (image: Uint8Array | ArrayBuffer) => {
 export const getImageBufferFromase64 = (content: string) =>
   Uint8Array.from(atob(content), (c) => c.charCodeAt(0))
 
-export const getImageBufferFromRemoteUrl = async (content: string) => {
-  const response = await fetch(content)
+export const getImageBufferFromRemoteUrl = async (content: string, options?: RequestInit) => {
```

Comment:
Hey @TCern you need to do the same for nodejs too. Because, This curretnyl effects only in `browser`. You need to make same changes inside `node.ts` file too